### ISSUE TITLE
Cached Renditions Documentum

### ISSUE THREAD

**Issue by eschwarzbeck** [tsgrp/HPI#1228]
If you use the `Set Rendition` action in Stage it will upload a new rendition to Documentum, however because we use the `Last Modified` header and check it against the `r_modify_date` whenever you view the document in stage it returns a 304 saying not modified and the old rendition is being displayed. Adding a new rendition in Documentum does not update the r_modify_date which is where the problem lies.

**Comment by gsteimer** [comment]
Yeah, since Documentum doesn't update the modified date, the fix for this would be to update the The workaround for this is to update the code to look to see if the PDF rendition has been modified.  The problem is that, for some back ends (ex: Alfresco), that is the modified date of the object, whereas for Documentum, it's not.

In the meantime, the only workaround is to add the type to the `content.getcontent.nocache.types` property and just skip the check to `304` entirely.

See also: https://github.com/tsgrp/HPI/issues/539

**Comment by gsteimer** [comment]
Note that if we implement this, the check to see if the PDF rendition has been modified should only take place if the user is requesting the PDF rendition.  If another file type is being requested, we should execute the code as it exists today.
